components/Table.vue

```diff
@@ -150,16 +136,14 @@
         }),
 
         mounted() {
-            this.defaultParams.selectedCol = this.selectedCol ? this.selectedCol : null;
-            this.defaultParams.order = this.order ? this.order.toLowerCase() : 'asc';
+            this.defaultParams.selectedCol = this.searchParams.selectedCol ? this.searchParams.selectedCol : null;
             this.defaultParams.paginate = this.paginate;
-
-            if (this.url) {
-                this.update(this.url);
-            } else {
-                this.internalData = this.data;
-                this.filteredData = this.data;
+            if (this.searchParams.order) {
```

Comment:
I think you should just loop through all the keys on the `searchParams` prop. That way you don't have to check keys that don't exist.